packages/slice-machine/src/features/environments/actions/getActiveEnvironment.ts

```diff
@@ -15,18 +15,27 @@ type GetActiveEnvironmentReturnType =
       error: unknown;
     };
 
-export async function getActiveEnvironment(): Promise<GetActiveEnvironmentReturnType> {
+export async function getActiveEnvironment(
+  isRetry = false,
+): Promise<GetActiveEnvironmentReturnType> {
   try {
-    const { activeEnvironment } =
+    const { activeEnvironment, error } =
       await managerClient.project.fetchActiveEnvironment();
 
+    if (error) {
+      const errorInstance = new Error(error.message);
+      errorInstance.name = error.name;
+      throw errorInstance;
```

Comment:
why do we need to build the error again?

Reply:
Since we don't throw from the manager, it's not really an error but just an object and so it cannot be used with the `isInvalidActiveEnvironmentError` if I don't rebuild it first